src/pages/WriteProductsPage.tsx

```diff
@@ -9,59 +10,72 @@ type FormData = {
 
 export const WriteProductsPage: React.FC = () => {
 	const { register, handleSubmit, formState: { errors } } = useForm<FormData>();
+	const [text, setText] = useState('投稿する？');
 
 	const onSubmit: SubmitHandler<FormData> = (data) => {
 		console.log('Submitted Data:', data);
 		alert('投稿できました！！！');
 	};
 
 	return (
-		<div>
-			<h1>Write Products Page</h1>
-			<form onSubmit={handleSubmit(onSubmit)}>
-				{/* タイトル入力フィールド */}
-				<div>
-					<label htmlFor='title'>タイトル</label>
-					<input
-						id='title'
-						{...register('title',{
-							required: 'タイトルは必須です'
-						})}
-						placeholder='タイトルを入力してください'
-					/>
-					{errors.title && <p style={{ color: 'red' }}>{errors.title.message}</p>}
-				</div>
+		<div className='flex justify-center bg-gray-100'>
+			<div className='w-full max-w-xl'>
+				<h1 className='text-lg font-bold text-gray-800 mt-4 mb-4 text-center'>Write Products Page</h1>
```

Comment:
```suggestion
				<h1 className='text-lg font-bold text-gray-800 my-4 text-center'>Write Products Page</h1>
```
top と bottom は y、left と right は x みたいに、左右どっちも同じ数値を指定するとかは、たいていのユーティリティクラスで xy に書き換えられるよ。